chore(profile): add Andrew Mazhaev

andrewMazhaev opened the pull request:
Closes #361

Add profile for Andrew Mazhaev

## Summary by CodeRabbit

- **Chores**
  - Added a new team profile for Andrew Mazhaev.

andrewMazhaev committed:
Update profile-submission.json

Signed-off-by: Andrew Mazhaev <mazhaev88@gmail.com>

andrewMazhaev committed:
Update profile-submission.json

Add profile for Andrew Mazhaev

Signed-off-by: Andrew Mazhaev <mazhaev88@gmail.com>

tiffmyaaw reviewed (changes requested):
Please rename your PR title and description based on our [Guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md#naming)

Also edit the PR description to [link your issue manually](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue).

Good luck!

andrewMazhaev committed:
Merge branch 'main' into andrewMazhaev-patch-1

Signed-off-by: Andrew Mazhaev <mazhaev88@gmail.com>

andrewMazhaev committed:
Fix JSON syntax: added comma and closing brackets

Resolved invalid JSON structure by:
- Adding a comma after my profile block
- Closing the team_profiles array and the root object

Signed-off-by: Andrew Mazhaev <mazhaev88@gmail.com>

andrewMazhaev commented:
Hi @tiffmyaaw,
I’ve made all the requested changes — updated the PR title, linked the issue manually, and fixed the JSON syntax ✅

This was my first time working with a PR like this, so it took me a bit of learning to figure everything out.
Thank you for the guidance — it really helped me get through it! Let me know if anything else is needed 🙌

tiffmyaaw commented:
> Hi @tiffmyaaw, I’ve made all the requested changes — updated the PR title, linked the issue manually, and fixed the JSON syntax ✅
> 
> This was my first time working with a PR like this, so it took me a bit of learning to figure everything out. Thank you for the guidance — it really helped me get through it! Let me know if anything else is needed 🙌

only the manual link to issue is missing on your PR description

andrewMazhaev commented:
It is done @tiffmyaaw

andrewMazhaev commented:
Okay @tiffmyaaw , tomorrow I may try to figure out what I did wrong. I've never done this before and interacting with code is not easy for me because I'm a designer not a programmer. Anyway, if it's mandatory, I can do it tomorrow.

andrewMazhaev commented:
I checked out other members' successful PRs and did the same in my PR. Check if everything is correct now? @tiffmyaaw

andrewMazhaev commented:
Also let me know why this needs to be done?

tiffmyaaw commented:
> Are you saying that I will be working mostly in GH and not Figma? That doesn't make sense @tiffmyaaw 

All of your work reports within our organization will need to be on GH
